apps/member-profile/app/routes/_profile.home.claim-swag-pack._index.tsx

```diff
@@ -104,20 +105,22 @@ export default function ClaimSwagPack() {
       </Modal.Header>
 
       <Suspense fallback={<LoadingState />}>
-        <Await resolve={inventoryPromise}>
-          {(inventory) => {
-            return inventory > 0 ? (
-              <ClaimSwagPackForm />
-            ) : (
-              <Modal.Description>
-                Unfortunately, we ran out of swag pack inventory. However, we're
-                restocking ASAP and you should be able to claim a pack in the
-                next 2-4 weeks. Sorry about any inconvenience and thank you for
-                your patience!
-              </Modal.Description>
-            );
-          }}
-        </Await>
+        {
```

Comment:
It seems that there is an extra pair of curly braces added here that weren't present before (and don't seem necessary)